docs/development/testing_and_dependencies.md

```diff
@@ -0,0 +1,13 @@
+## Dependency management
+[Dep](https://github.com/golang/dep) is the depedency management tool. In order to add a new package dependency to the project, you can perform `dep ensure -add <PACKAGE>` or edit the `Gopkg.toml` file and append the package along with the version you want to use as a new `[[constraint]]`.
```

Comment:
s/[Dep](https://github.com/golang/dep) is the depedency management tool./We use [Dep](https://github.com/golang/dep) to manage golang dependencies./

If someone wants to add dependencies we should check license etc in order to ensure we don't depend on a license we don't want